src/devices/Nrf24l01/Nrf24l01.cs

```diff
@@ -712,11 +712,14 @@ internal byte ReadChannel()
         #endregion
 
         #region sensor operation
-        internal void Write(Span<byte> writeData)
+        internal void Write(ReadOnlySpan<byte> writeData)
         {
             Span<byte> readBuf = stackalloc byte[writeData.Length];
+            Span<byte> writeBuf = stackalloc byte[writeData.Length];
 
-            _sensor.TransferFullDuplex(writeData, readBuf);
+            writeData.CopyTo(writeBuf);
```

Comment:
can you just directly pas writeData to TransferFullDuplex?

Reply:
VS will report wrong

Reply:
VS will report wrong

Reply:
Interesting, that sounds like a bug in the Spi abstraction: https://github.com/dotnet/iot/blob/master/src/System.Device.Gpio/System/Device/Spi/SpiDevice.cs#L51 they should be using ReadOnlySpan

Reply:
@ZhangGaoxing I've sent PR with the fix: https://github.com/dotnet/iot/pull/240

Reply:
@ZhangGaoxing this is merged now, you can update the PR